### ISSUE TITLE
Repository Management 

### ISSUE THREAD

**Issue by shivangdubey** [shivangdubey/HacktoberFest2020#100]
Currently we have given Language folders:
1) C
2) C++
3) Python
4) Java
5) Images
Kindly arrange all the codes accordingly and give folders a better formatting.

**Comment by Grasstown** [comment]
Mind if I work on this? Is it just renaming the folders and organizing files inside them?

**Comment by shivangdubey** [comment]
Yes, and also distribute the codes lying outside the folders accordingly.

**Comment by Darshil-Papalkar** [comment]
@shivangdubey can I work on the issue

**Comment by shivangdubey** [comment]
> @shivangdubey can I work on the issue

Yes.

**Comment by shivangdubey** [comment]
Fix the Leetcode folder. The contents are not properly named and the folder doesn't contain any readme or .md file.

**Comment by Ayan1089** [comment]
Can I work on this issue?
